src/OpenLoco/Map/Tile.h

```diff
@@ -81,6 +81,7 @@ namespace OpenLoco::Map
         uint8_t baseZ() const { return _base_z; }
         uint8_t clearZ() const { return _clear_z; }
 
+        uint8_t direction() const { return _type & 0x03; }
```

Comment:
although this is mostly true for every element we haven't defined it as a property of every type.